MmSupervisorPkg/Core/Misc/MemoryAttributesTable.c

```diff
@@ -352,6 +352,75 @@ SmmCreateImageRecordInternal (
   return Status;
 }
 
+EFI_STATUS

```

Comment:
Something like this should have a function description block.

Reply:
oops, missed that,

Reply:
Updated